site/content/internal/writing-a-blog-post.md

```diff
@@ -77,6 +77,7 @@ The steps below outline the process involved in creating the blog post file from
     <some more content>
     ```
 
-4. Write your blog post. 
-5. Submit a pull request to https://github.com/mattermost/mattermost-developer-documentation and assign two dev reviews and an editor review from @amyblais or @justinegeffen.
-6. Once merged it should show up on [developers.mattermost.com/blog](https://developers.mattermost.com/blog) within 10-15 minutes. When it shows up, post about it in the Developers channel on community.mattermost.com.
+4. Write your blog post.
+5. (Optional) If you wrote the blog post with someone else, you can also add a second author by adding `author_2`, `github_2` and `community_2` to the front matter.
```

Comment:
Not sure if "front matter" could be more specific or if it's correct as is.

Reply:
I've added a link to the Hugo docs .